app/models/post.rb

```diff
@@ -1,4 +1,21 @@
 class Post < ApplicationRecord
   belongs_to :user
   has_many :comments
+
+  # Calculate the score of the post for the purpose of selecting trending posts
+  # formula is log_10(x) + y * (submission_time - epoch)/45000
+  # where x = max{number of likes + number of comments, 1}, epoch = 1970/1/1,
+  # time difference is in seconds, y = 1 if there is at least one like or
+  # comment, and 0 otherwise
+  def compute_trending_score
```

Comment:
Hmm do you think we can improve the readability of this method somehow?

Reply:
What do x and y represent? Maybe we can rename them :)

Reply:
Hmm x->activities, y->sign?

Reply:
Maybe you can do something like
```
mock_num_likes = 5
num_activities = mock_num_likes + self.comments.count
sign = num_activities > 0 ? 1 : 0
num_activities = [num_activities, 1].max
```

And assign 45000 to a descriptive variable in a similar manner.